docs/gathered-data.md

```diff
@@ -598,35 +598,50 @@ Response see:
 
 ## MostRecentMetrics
 
-gathers cluster Federated Monitoring metrics.
+Collects cluster Federated Monitoring metrics.
 
 The GET REST query to URL /federate
 Gathered metrics:
+  - `virt_platform`
+  - `etcd_object_counts`
+  - `cluster_installer`
+  - `vsphere_node_hw_version_total`
+  - namespace CPU and memory usage
+  - `console_helm_installs_total`
+  - `console_helm_upgrades_total`
+  - `console_helm_uninstalls_total`
+  - followed by at most 1000 lines of `ALERTS` metric
 
-	virt_platform
-	etcd_object_counts
-	cluster_installer
-	vsphere_node_hw_version_total
-	namespace CPU and memory usage
-	console_helm_installs_total
-	console_helm_upgrades_total
-	console_helm_uninstalls_total
-	followed by at most 1000 lines of ALERTS metric
-
-* Location in archive: config/metrics
-* See: docs/insights-archive-sample/config/metrics
-* Id in config: clusterconfig/metrics
-* Since version:
-  - "etcd_object_counts": 4.3+
-  - "cluster_installer": 4.3+
-  - "ALERTS": 4.3+
-  - "namespace:container_cpu_usage_seconds_total:sum_rate": 4.5+
-  - "namespace:container_memory_usage_bytes:sum": 4.5+
-  - "virt_platform metric": 4.6.34+, 4.7.16+, 4.8+
-  - "vsphere_node_hw_version_total": 4.7.11+, 4.8+
-  - "console_helm_installs_total": 4.11+
-  - "console_helm_upgrades_total": 4.12+
-  - "console_helm_uninstalls_total": 4.12+
+### API Reference
+None
+
+### Sample data
+- [docs/insights-archive-sample/config/metrics](./insights-archive-sample/config/metrics)
```

Comment:
Not related to this PR, but I think the target `make docs` for generating the `gathered-data.md` is currently not working correctlym because it removes the link here.

Reply:
It will remove it, 'cause the `make docs` wasn't migrated.